### DISCUSSION TITLE
How to enable debug-logging?

### DISCUSSION THREAD

**Discussion by eecheng87** [dragonflydb/dragonfly#425] [Q&A] [answered]
Hi, all

I try to enable debug logging as many as possible (e.g., `DVLOG`) when running dragonfly. How can I achieve it? 

Thanks in advance!

**Comment by romange** [comment, marked as answer]
in what file? you can use `--vmodule=main_service=1,set_family=2` etc.

**Reply by eecheng87** [reply]
Thanks! That is what I want.
